themes/aframe/layout/docs.ejs

```diff
@@ -1,39 +1,76 @@
+        <%- version %>
+      </option>
+    <% }); %>
+  </select>
+
   <div class="list">
-    <% var activeSection = '' %>
-    <% site.pages.find({parent_section: 'docs'}).sort('section_order').each(function (section) { %>
+    <% docs_version_filter(site.pages, activeVersion).find({parent_section: 'docs'})
+                                                             .sort('section_order')
+                                                             .each(function (section) { %>
       <% if (page.type === section.type) { activeSection = section } %>
-      <h3 class="section-title docs-section-title<%- page.type === section.type ? ' section-title-active' : ' section-title-inactive' %>"><a href="<%- page_url('docs/' + section.type + '/') %>" class="section-title-link<%= page.type === section.type ? ' section-title-link-active' : ' section-title-link-inactive' %>"><%= section.section_title %></a></h3>
 
+      <!-- Section title. -->
+      <h3 class="section-title docs-section-title<%- page.type === section.type ? ' section-title-active' : ' section-title-inactive' %>">
+        <a href="<%- page_url('docs/' + activeVersion + '/' + section.type + '/') %>"
+           class="section-title-link<%= page.type === section.type ? ' section-title-link-active' : ' section-title-link-inactive' %>">
+          <%= section.section_title %>
+        </a>
+      </h3>
+
+      <!-- Section links. -->
       <ol class="subnav section-subnav<%= section.show_guide ? ' numbered-subnav' : '' %>">
-        <% site.pages.find({type: section.type}).sort('order').each(function (p) { %>
+        <% docs_version_filter(docs_nav_sort(site.pages.find({type: section.type})),
+                               activeVersion).forEach(function (p) { %>
           <li class="subnav-item <%= page.title === p.title ? ' subnav-item-current' : '' %>">
-            <a href="<%- page_url(p.path) %>" class="subnav-link<%= page.title === p.title ? ' current' : '' %>"><%= p.title %></a>
+            <a href="<%- page_url(p.path) %>"
+               class="subnav-link<%= page.title === p.title ? ' current' : '' %>">
+              <%= p.title %>
+            </a>
           </li>
         <% }) %>
       </ol>
     <% }) %>
   </div>
-
 </aside>
 
+<!-- Documentation. -->
 <div class="content content--structured content--<%- page.type %> copy">
   <div class="copy__wrap">
-
-    <p class="page__section"><a href="<%- page_url(activeSection.path) %>" class="borderless-link"><%= activeSection.section_title %></a></p>
+    <p class="page__section">
+      <a href="<%- page_url(activeSection.path) %>" class="borderless-link">
+        <%- activeVersion %> &rsaquo; <%= activeSection.section_title %>
+      </a>
+    </p>
+    <% if (activeVersion === 'master') { %>
+      <%- markdown("> **Note:** The `master` version of the documentation tracks unstable development happening on [A-Frame's master branch](https://github.com/aframevr/aframe)") %>
```

Comment:
the blockquote style has a green border, which makes this look like a success message (IMO)